assets/shaders/custom_material.frag

```diff
@@ -1,11 +1,18 @@
 #version 450
+layout(location = 0) in vec2 v_Uv;
 
 layout(location = 0) out vec4 o_Target;
 
 layout(set = 1, binding = 0) uniform CustomMaterial {
     vec4 Color;
 };
 
+// Naga GLSL does not support the sampler2D type, but only a texture2D + sampler combination
+// See https://github.com/gfx-rs/naga/issues/1012
```

Comment:
ha coming back at this, I'm torn... I don't expect the average Bevy user to know what Naga is. Should that be explained somewhere? But the example is really not the place for that

Reply:
There's a link to the issue, so they can find the repo from that :)

Reply:
I became acquainted with naga only because naga didn't support everything in the wgsl spec.  Naga's growing pains have been a pain point for me multiple times when using bevy.  It definitely wasn't easy to figure out where the issues I was facing were coming from and what the solutions were.  Something in the bevy project linking to naga for shader issues (issue template?) and a good documentation source for wgsl would be _really_ helpful.

At the very least, we should clearly indicate here in the example that the GLSL used for writing shaders in bevy is a specific _flavor_ of GLSL here, since this is probably the first source of documentation someone would encounter for using GLSL in their project.

Reply:
I've removed this comment. Imo examples are a place to show _how_ to do something, not a place to enumerate _how not_ to do something.

If we want to document Naga limitations, we should do it somewhere else.